pkg/kube/controllers/statefulset/statefulset_rollout_controller.go

```diff
@@ -0,0 +1,68 @@
+package statefulset
+
+import (
+	"context"
+
```

Comment:
remove newline please
we also group imports in 
- standard lib
- third party
- project

Reply:
This conflicts with another comment https://github.com/cloudfoundry-incubator/cf-operator/pull/692#discussion_r352036555 by @manno 
which says:
* standard lib
* third party
* k8s
* project

What is correct?